### PR TITLE
Some low hanging cleanups for Insight

### PR DESCRIPTION
**Pull request by rarila**

### CODE REVIEW

File: src/YamlUpdater.php [hunk 1]
```diff
@@ -29,12 +29,6 @@ class YamlUpdater
     private $lines = 0;
 
     /**
-     * The YAML string of the updated file
-     * @var string
-     */
-    private $yml = '';
-
-    /**
```
Comment (GawainLynch) [inline src/YamlUpdater.php]: Ooops...

Comment (rarila) [inline src/YamlUpdater.php]: Ooops… this has to stay or Ooops… why was this here?

Comment (GawainLynch) [inline src/YamlUpdater.php]: > Ooops… this has to stay or Ooops… why was this here?

As in "Oooops, I forgot to remove that"

Comment (rarila) [inline src/YamlUpdater.php]: Big Insight is watching you!

**Comment by bobdenotter** [comment]
Oops.

tumblr_mheqkagopf1qihp55o1_250

**Comment by rarila** [comment]
Ooops, never do that again!

**Comment by GawainLynch** [comment]
I get landed with Britney Spears :cry:
